spring-data-mongodb/src/main/java/org/springframework/data/mongodb/gridfs/GridFsTemplate.java

```diff
@@ -15,14 +15,12 @@
  */
 package org.springframework.data.mongodb.gridfs;
 
-import static org.springframework.data.mongodb.core.query.Query.*;
```

Comment:
Please do not reformat the imports. Please find the import settings [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide).